packages/wonder-blocks-core/util/types.js

```diff
@@ -135,48 +133,48 @@ export type AriaProps = {|
 |};
 
 type MouseEvents = {|
-    onMouseDown?: (e: SyntheticMouseEvent<*>) => void,
-    onMouseUp?: (e: SyntheticMouseEvent<*>) => void,
-    onMouseMove?: (e: SyntheticMouseEvent<*>) => void,
-    onClick?: (e: SyntheticMouseEvent<*>) => void,
-    onDoubleClick?: (e: SyntheticMouseEvent<*>) => void,
-    onMouseEnter?: (e: SyntheticMouseEvent<*>) => void,
-    onMouseLeave?: (e: SyntheticMouseEvent<*>) => void,
-    onMouseOut?: (e: SyntheticMouseEvent<*>) => void,
-    onMouseOver?: (e: SyntheticMouseEvent<*>) => void,
-    onDrag?: (e: SyntheticMouseEvent<*>) => void,
-    onDragEnd?: (e: SyntheticMouseEvent<*>) => void,
-    onDragEnter?: (e: SyntheticMouseEvent<*>) => void,
-    onDragExit?: (e: SyntheticMouseEvent<*>) => void,
-    onDragLeave?: (e: SyntheticMouseEvent<*>) => void,
-    onDragOver?: (e: SyntheticMouseEvent<*>) => void,
-    onDragStart?: (e: SyntheticMouseEvent<*>) => void,
-    onDrop?: (e: SyntheticMouseEvent<*>) => void,
+    onMouseDown?: (e: SyntheticMouseEvent<*>) => mixed,
+    onMouseUp?: (e: SyntheticMouseEvent<*>) => mixed,
+    onMouseMove?: (e: SyntheticMouseEvent<*>) => mixed,
```

Comment:
Thanks for changing this.  I'll try to be more thoughtful about dev ergonomics when typing things in the future.